lib/datadog/core/remote/dispatcher.rb

```diff
@@ -7,8 +7,8 @@ module Remote
       class Dispatcher
         attr_reader :receivers
 
-        def initialize
-          @receivers = []
+        def initialize(receivers)
+          @receivers = receivers
```

Comment:
The main difference here is that before the receiver list is guaranteed to be owned by the `Dispatcher` instance, whereas it is not anymore afterwards.

The only place it is used though is upon `Client#initialize` so the change is fair.